src/pages/templates/edit-overpanel.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react'
+import '../../sass/example-page/example-page.scss'
+
+export default () => (
+
+	<div className="overpanel">
+		<header className="title-bar">
+			<div className="title-content">
+				<div>
+					<h2>Edit Address</h2>
+					<p>Manage the address tied to your membership</p>
+				</div>
+				<button className="button button--secondary"><i className="dashing-icon dashing-icon--close"></i> Close</button>
+			</div>
+		</header>
+
+    <section className="app">
```

Comment:
Small thing. Aren't we killing the `app` wrapper?

Reply:
Oh yeah! I'll remove that. Pretty sure it isn't doing anything.